### PR TITLE
feat(WebexMember): display if user is host

### PR DESCRIPTION
**Pull request by karinasigartau0798**
Updated WebexMember to display if user is the host of a meeting.

https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-222973

### CODE REVIEW

File: src/components/WebexMember/WebexMember.scss [hunk 1]
```diff
@@ -16,7 +16,12 @@
     text-overflow: ellipsis;
   }
 
-  .subtext {
+  .organization {
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.scss:19]: `.subtext` needs to stay (used for "You"), `.presenter` needs to go

Comment (cipak) [reply src/components/WebexMember/WebexMember.scss:19]: Ok I see now that `.subtext` is still present... please remove `.presenter`

File: src/components/WebexMember/WebexMember.jsx [hunk 2]
```diff
@@ -22,23 +23,32 @@ export default function WebexMember({
   personID,
   displayStatus,
 }) {
-  const {firstName, lastName} = usePerson(personID);
+  const {firstName, lastName, orgID} = usePerson(personID);
   const me = useMe();
   const members = useMembers(destinationID, destinationType);
   const member = members
     .find((itemMember) => itemMember.ID === personID);
+  const organization = useOrganization(orgID);
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.jsx:31]: Seems like there are some changes from external user story mingled in here?
Can you make sure the changes only relate to "host"?

File: src/data/memberships.js [hunk 3]
```diff
@@ -73,20 +80,31 @@ export default {
...
       {
         ID: 'user3',
         orgID: 'org1',
         muted: true,
         sharing: false,
         inMeeting: true,
+        host: true,
+      },
+      {
+        ID: 'user5',
```
Comment (lalli-flores) [inline src/data/memberships.js:102]: I think this change is related to external org users and should be part of a different PR

**Review by lalli-flores — CHANGES_REQUESTED** [review]
The host/ roles changes look good, but there are changes related to external org users that shouldn't be part of this PR. If you could remove those, then we should be good to go

File: src/components/WebexMember/WebexMember.jsx (hunk 2 above)
Comment (karinasigartau0798) [reply src/components/WebexMember/WebexMember.jsx]: Done.

File: src/data/memberships.js (hunk 3 above)
Comment (karinasigartau0798) [reply src/data/memberships.js]: You're right.
Done.

**Review by lalli-flores — APPROVED** [review]
Awesome! You even indirectly fixed a UX bug from having "You" and "presenter" roles in different lines 🥳
